src/site/_data/site.js

```diff
@@ -53,7 +53,7 @@ module.exports = {
       NAVIGATION_TYPE: 'dimension6',
       COLOR_SCHEME_PREFERENCE: 'dimension7',
     },
-    version: 10,
+    version: 11,
```

Comment:
What's this for, out of interest?

Reply:
Bumping the analytics version is useful in case you want to compare performance before and after. This corresponds to a custom dimension (`dimenson5`) sent on all analytics beacons.